src/pages/_app.tsx

```diff
@@ -55,29 +61,31 @@ export default function App({ Component, pageProps, layoutProps, host }) {
 		<>
 			<SSRProvider>
 				<ErrorBoundary FallbackComponent={DevDotClient}>
-					<DeviceSizeProvider>
-						<AllProductDataProvider>
-							<CurrentProductProvider currentProduct={currentProduct}>
-								<CodeTabsProvider>
-									<HeadMetadata {...pageProps.metadata} host={host} />
-									<LazyMotion
-										features={() =>
-											import('lib/framer-motion-features').then(
-												(mod) => mod.default
-											)
-										}
-										strict={process.env.NODE_ENV === 'development'}
-									>
-										<Layout {...allLayoutProps} data={allLayoutProps}>
-											<Component {...pageProps} />
-										</Layout>
-										<Toaster />
-										{showProductSwitcher ? <PreviewProductSwitcher /> : null}
-									</LazyMotion>
-								</CodeTabsProvider>
-							</CurrentProductProvider>
-						</AllProductDataProvider>
-					</DeviceSizeProvider>
+					<SessionProvider session={session}>
```

Comment:
Just to confirm: is adding this as a wrapper around most of the existing items here the only change in this area? The changes above are obvious, but GitHub isn't diff-ing this party very nicely 😆

Reply:
Yup! You can hide whitespace changes on a diff which helps in this case: https://github.com/hashicorp/dev-portal/pull/717/files/e849d0e177b466e3bff59dedc320d9a86422a98c?w=1

Reply:
OOF I forgot I turned that off recently when we converted from `SPACE` to `TAB` indentation 🤣